### ISSUE TITLE
Bindgen ERROR: unsupported type `CXTypeKind(119)`

### ISSUE THREAD

**Issue by MabezDev** [fortanix/rust-mbedtls#61]
```
$ clang --version

clang version 8.0.1 (tags/RELEASE_801/final)
Target: x86_64-pc-linux-gnu
Thread model: posix
InstalledDir: /usr/bin
```

Is there a specific version of clang required to build this project?

**Comment by jethrogb** [comment]
Unfortunately the version of bindgen used requires clang 3.8 or older right now. We're working on fixing this.

Closing as duplicate of #5.

**Comment by jethrogb** [comment]
Final PR: #152

**Comment by jethrogb** [comment]
Fixed in #152
